lighthouse-core/lib/i18n/i18n.js

```diff
@@ -265,8 +265,12 @@ function _resolveIcuMessageInstanceId(icuMessageInstanceId, locale) {
 }
 
 /**
- * @param {LH.Result} lhr
+ * Recursively walk the input object, looking for property values that are
+ * string references and replace them with their localized values. Primarily
+ * used with the full LHR as input.
+ * @param {*} lhr
```

Comment:
why not `{LH.Result}` ?

Reply:
> why not `{LH.Result}` ?

I pass in the config now to get a translated one. I can rename the param to make that more obvious